### PR TITLE
06 done too - mrcll

### PR DESCRIPTION
**Pull request by remrcll**
pls check.

### CODE REVIEW

File: 06-PB-ternary-exercises/06-mrcll-solution.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+"use strickt"
+
```
Comment (martl) [inline 06-PB-ternary-exercises/06-mrcll-solution.js:2]: use strict (no ck)

File: 06-PB-ternary-exercises/06-mrcll-solution.js [hunk 2]
```diff
@@ -9,6 +11,7 @@ let vizsla = isDog;
 // 2. Declare a variable named speedCheck. If speedlimit is 50km/h, and your speed is above that, print 'you're going too fast!'. If speed is lower than 50km/h, print 'You're driving below the speed limit, Oma'.
 
 let speedCheck = 50;
+let message;
 
 speedCheck < 60 ? message = "you're going too fast!" : message = "You're driving below the speed limit, Oma";
```
Comment (martl) [inline 06-PB-ternary-exercises/06-mrcll-solution.js:16]: you can also turn it around 

message = speedCheck < 60 ? "you're going too fast!" : "You're driving below the speed limit, Oma";

File: 06-PB-ternary-exercises/06-mrcll-solution.js [hunk 3]
```diff
@@ -20,6 +23,7 @@ console.log(message);
 
 let age = 16;
 
+
 age < 15 ? message = "serve beer" : message = "serve butter beer";
```
Comment (martl) [inline 06-PB-ternary-exercises/06-mrcll-solution.js:27]: you can also turn it around 

message = speedCheck < 60 ? "you're going too fast!" : "You're driving below the speed limit, Oma";

File: 06-PB-ternary-exercises/06-mrcll-solution.js [hunk 4]
```diff
@@ -41,4 +45,4 @@ console.log(isStudent);
 let okMarie = 'cake';
 (okMarie == 'cake') ? console.log('let them eat cake!'): console.log('oh, bother');
 
-(okMarie == 'vegetabe') ? console.log('let them eat cake!'): console.log('oh, bother');
\ No newline at end of file
+(okMarie == 'vegetabe') ? console.log('let them eat cake!'): console.log('oh, bother');
```
Comment (martl) [inline 06-PB-ternary-exercises/06-mrcll-solution.js:48]: always use === instead of ==

File: 06-PB-ternary-exercises/06-mrcll-solution.js (hunk 2 above)
Comment (remrcll) [reply 06-PB-ternary-exercises/06-mrcll-solution.js:16]: This ternary twists my mind out completely :(

**Pull request closed without merging**